### PR TITLE
Convert Card View demo to TS

### LINKED ISSUE #9362: Convert card view demo to TS
Patterns -> Card view demo

### PR DESCRIPTION
**Pull request by jenny-s51**
**What**: Towards #9124, Closes #9362

**Additional issues**: Waiting on #9322. Will rebase once #9322 is merged.

### CODE REVIEW

**Review by kmcfaul — APPROVED** [review]

**Comment by wise-king-sullyman** [comment]
Code looks good, but unless I'm misunderstanding something (not unlikely) shouldn't this update the markdown file to point to this new tsx file?

**Review by jenny-s51 — COMMENTED** [review]
@wise-king-sullyman @tlabaj yes, thanks! Updated

**Review by wise-king-sullyman — APPROVED** [review]

**Comment by wise-king-sullyman** [comment]
Looks good to me other than the docs build failing CI.

File: packages/react-core/src/demos/Card/examples/CardView.tsx [hunk 1]
```diff
@@ -0,0 +1,574 @@
...
+import swaggerIcon from './assets/camel-swagger-java_200x150.png';
+import azureIcon from './assets/FuseConnector_Icons_AzureServices.png';
+import restIcon from './assets/FuseConnector_Icons_REST.png';
+import EllipsisVIcon from '@patternfly/react-icons/dist/esm/icons/ellipsis-v-icon';
+import DashboardWrapper from '@patternfly/react-core/src/demos/examples/DashboardWrapper';
+
+export const CardViewBasic: React.FunctionComponent = () => {
+  const totalItemCount = 10;
+
+  const [cardData, setCardData] = React.useState(data);
```
Comment (tlabaj) [inline packages/react-core/src/demos/Card/examples/CardView.tsx:59]: looks like you are missing the import for `data`

**Review by tlabaj — APPROVED** [review]

**Pull request merged**